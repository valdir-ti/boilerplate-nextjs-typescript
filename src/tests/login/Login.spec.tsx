import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'

import Login from '../../pages/login'

describe('<Login />', () => {
  it('should render a heading', () => {
    render(<Login />)
    expect(screen.getByRole('heading', { name: /Login/i })).toBeInTheDocument()
  })

  it('should match a snapshot', () => {
    const { container } = render(<Login />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
