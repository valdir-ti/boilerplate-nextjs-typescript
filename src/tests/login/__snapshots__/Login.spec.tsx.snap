// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> should match a snapshot 1`] = `
<div>
  <h3>
    Login
  </h3>
</div>
`;
